plugin/src/main/java/net/dzikoysk/funnyguilds/config/PluginConfiguration.java

```diff
@@ -1125,15 +1121,13 @@ public void loadProcessedProperties() {
 
         this.tntProtection.time.passingMidnight = this.tntProtection.time.startTime.getTime().isAfter(this.tntProtection.time.endTime.getTime());
 
-        if (!"v1_8_R1".equals(Reflections.SERVER_VERSION) && !"v1_8_R3".equals(Reflections.SERVER_VERSION)) {
+        if (!"v1_8_R3".equals(Reflections.SERVER_VERSION)) {
```

Comment:
Seems unrelated to the main purpose of the PR?

Reply:
It's not only this PR has unrelated changes so don't mind about that. Here it's normal

Reply:
![](https://i.pinimg.com/736x/b0/61/1a/b0611ac1baf455d511802d10f2c47713.jpg)

Reply:
Yes